### ISSUE TITLE
Make Google Play services error dialog optional

### ISSUE THREAD

**Issue by sal0max** [balwinderSingh1989/androidBestLocationTracker#4]
Thanks for this great library. It is the only one I could find which provides a convenient way to access Android GPS data without the need of Google Play services (FusedLocationProvider).

I tested your library with great success on an emulator without Google Play services. It works great! However, there's still this message dialog popping up, like in this screenshot:
![Screenshot_1656887882](https://user-images.githubusercontent.com/423373/177059630-3f64b880-b6ff-4cb4-93ee-5d3f2547744d.png)

I guess this is because of [this](https://github.com/balwinderSingh1989/androidBestLocationTracker/blob/f11faede8a3a1f2cf0bce9b1c6e719ce647022b5/bestlocationstrategy/src/main/java/com/location/bestlocationstrategy/LocationUtils.kt#L30) code:
```kotlin
if (googleApiAvailability.isUserResolvableError(status)) {
   googleApiAvailability.getErrorDialog(context as Activity?, status, 2404).show()
}
```

**Is there a chance you could make this error dialog optional?**

It could confuse users and realistically, they can't change anything on the situation or even chose actively not to use Google services. And as mentioned: Despite the error message, location fetching is working perfectly fine without Google services.

**Comment by balwinderSingh1989** [comment]
Hi @sal0max

Thank you for reporting this issue. Have addressed the same in the latest version 2.0.3. Please update and check again :) 

Regards,
Balwinder

**Comment by sal0max** [comment]
Can confirm: Working as expected, now. Thanks a lot! :)
